src/database/mysql/mysql-upgrade.xml

```diff
@@ -0,0 +1,60 @@
+<upgrade>
+    <version from="1" to="2">
+        <script>ALTER TABLE `mt_cds_object` CHANGE `location` `location` BLOB NULL DEFAULT NULL</script>
+        <script>ALTER TABLE `mt_cds_object` CHANGE `metadata` `metadata` BLOB NULL DEFAULT NULL</script>
+        <script>ALTER TABLE `mt_cds_object` CHANGE `auxdata` `auxdata` BLOB NULL DEFAULT NULL</script>
+        <script>ALTER TABLE `mt_cds_object` CHANGE `resources` `resources` BLOB NULL DEFAULT NULL</script>
+        <script>ALTER TABLE `mt_autoscan` CHANGE `location` `location` BLOB NULL DEFAULT NULL</script>
+    </version>
+    <version from="2" to="3">
+        <script>ALTER TABLE "ALTER TABLE `mt_autoscan` CHANGE `scan_mode` `scan_mode` ENUM( 'timed', 'inotify' ) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL</script>
+        <script>ALTER TABLE `mt_autoscan` DROP INDEX `mt_autoscan_obj_id`, ADD UNIQUE `mt_autoscan_obj_id` ( `obj_id` )</script>
+        <script>ALTER TABLE `mt_autoscan` ADD `path_ids` BLOB AFTER `location`</script>
+    </version>
+    <version from="3" to="4">
+        <script>ALTER TABLE `mt_cds_object` ADD `service_id` varchar(255) default NULL</script>
+        <script>ALTER TABLE `mt_cds_object` ADD KEY `cds_object_service_id` (`service_id`)</script>
+    </version>
+    <version from="4" to="5" remark="move to metadata">
+        <script>
+        CREATE TABLE `mt_metadata` (
+            `id` int(11) NOT NULL auto_increment,
+            `item_id` int(11) NOT NULL,
+            `property_name` varchar(255) NOT NULL,
+            `property_value` text NOT NULL,
+            PRIMARY KEY `id` (`id`),
+            KEY `metadata_item_id` (`item_id`),
+            CONSTRAINT `mt_metadata_idfk1` FOREIGN KEY (`item_id`) REFERENCES `mt_cds_object` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+        ) ENGINE=MyISAM CHARSET=utf8
+        </script>
+    </version>
+    <version from="5" to="6" remark="add config value table">
+        <script>
+        CREATE TABLE `grb_config_value` (
+            `item` varchar(255) primary key,
+            `key` varchar(255) NOT NULL,
+            `item_value` varchar(255) NOT NULL,
+            `status` varchar(20) NOT NULL)
+            ENGINE=MyISAM CHARSET=utf8
+        </script>
+        <script>CREATE INDEX grb_config_value_item ON grb_config_value(item)</script>
+    </version>
+    <version from="6" to="7">
+        <script>DROP TABLE mt_cds_active_item</script>
+    </version>
+    <version from="7" to="8" remark="part_number">
```

Comment:
Maybe just version_number and work on the assumption you need apply previous versions? Every update we have is +1 anyway.

Reply:
It's documentary anyway